namespaces/live.cloud-platform.service.justice.gov.uk/laa-means-prototype/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: laa-means-prototype-admin
+  namespace: laa-means-prototype
+subjects:
+  - kind: Group
+    name: "github:all-org-members"
```

Comment:
Can you mention the specific github team who is responsible for this repo

Reply:
Done